biztalk/dev-center/mvp-and-community-blogs.md

```diff
@@ -58,12 +58,12 @@ Aram has been building software since 2001. And when not creating software, Aram
 Contains wealth of information related to Microsoft Integration technologies mainly around BizTalk Server. There are over 800 blog articles written by Microsoft MVP's and  the BizTalk360 team. 
 
 ## Edward Kuo 
```

Comment:
Please remove Edward Kuo altogether as I couldn't find any other blog or Twitter content. No point including this entry, if there are no links to follow.